runtime/block_welford_outer.cu

```diff
@@ -56,7 +56,7 @@ namespace impl {
 // registers than just returing the output. Results would vary
 // depending on compiler versions, but it seems safer to return outputs
 // as a new value.
-template <int NumVals, typename DataType, int BDIMX, int BDIMY>
+template <bool ALIGNED, int NumVals, typename DataType, int BDIMX, int BDIMY>
```

Comment:
nit: use variable names of the same style
```suggestion
template <bool ALIGNED, int NUM_VALS, typename DATA_TYPE, int BDIMX, int BDIMY>
```

Reply:
Hmm, maybe not a blocker for this PR, but naming variables with different style is very ugly, and we have a lot of it....

Reply:
Yeah, right, haven't paid much attention to it. I'd prefer the camel case like `Aligned`, `NumVals`, etc. That's generally the case in our `csrc` codebase. I'll change `ALIGNED` to `Aligned`.